packages/web-app-files/src/helpers/resource/actions/upload.ts

```diff
@@ -210,6 +210,13 @@ export class ResourcesUpload extends ConflictDialog {
     const uploadSizeSpaceMapping = uppyResources.reduce((acc, uppyResource) => {
       let targetUploadSpace
 
+      const existingFile = this.currentFiles.find(
+        (c) => !uppyResource.meta.relativeFolder && c.name === uppyResource.name
+      )
+      if (existingFile?.size >= uppyResource.data.size) {
+        return acc
+      }
```

Comment:
Hmm - wouldn't it be more accurate to substract the difference from acc in this case? 

Can't we do something like this?
```javascript
if(existingFile?.size) {
  return acc - existingFile.size + uppyResource.data.size
}
```

Reply:
Ohh yes, makes much more sense actually! Thx